App.js

```diff
@@ -15,7 +15,7 @@ import FavoritesScreen from './src/screens/FavoritesScreen';
 import AddSiteScreen from './src/screens/AddSiteScreen';
 import MoreScreen from './src/screens/MoreScreen';
 
-const store = createStore(reducers, {}, applyMiddleware(ReduxThunk));
+const store = createStore(reducers, {}, compose(applyMiddleware(ReduxThunk)));
```

Comment:
We'll need the `compose` helper anyway when we add other pieces here, so we might as well add it now.